src/common/com/intellij/plugins/haxe/lang/parser/HaxePsiBuilder.java

```diff
@@ -68,7 +69,8 @@ public HaxePsiBuilder(@NotNull Project project,
                         @NotNull Lexer lexer,
                         @NotNull LighterLazyParseableNode chameleon,
                         @NotNull CharSequence text) {
-    super(project, parserDefinition, lexer, chameleon, text);
+    // FIXME: https://github.com/JetBrains/intellij-community/commit/bf92a4bfea21ac1269017a52a347ab32c1a87323#r105729752
```

Comment:
`super()` call should work unchanged, see commit discussion for clarification

Reply:
fixed in https://github.com/JetBrains/intellij-community/commit/be794f31c519a7188f4164c1b3f008e9762c0a7d: wait for a release tag, bump plugin SDK and roll back `(ASTNode)` cast